### ISSUE TITLE
Request for an Option to Disable the Full-Screen 'X' Exit Button

### ISSUE THREAD

**Issue by HornyPrivateGamer** [brave/brave-browser#42394]
### Platforms

all, Windows

### Description

For websites like YouTube, after entering full-screen mode on Windows, this grey X dot appears. However, this does not seem to be the case on macOS. Which has proven having "X" button will result in a worse user experience. 

It’s very distracting, much like the old “Esc to exit full screen” message, which we now thankfully have an option to disable. If we really want to exit full-screen mode, we can easily do it manually or by pressing "F" on YouTube. Having the ability to disable this X button should also be an option.

I’m sincerely asking for this feature to be added, similar to the option to disable the “Esc to exit full screen” message. Many users don’t like it, and this has been reported widely on platforms like Reddit, the Brave community, and Google forums.

Here are some examples (and there are plenty more):

https://community.brave.com/t/how-to-disable-this-fullscreen-x-exit-button/144391
https://support.google.com/chrome/thread/2757504/remove-black-circle-x-button-in-fullscreen-mode?hl=en
https://www.reddit.com/r/BoosteroidCommunity/comments/1d3fkmp/disable_exit_full_screen_button_for_chrome/
https://www.reddit.com/r/Diepio/comments/97nlnj/how_to_remove_chromes_x_dot_which_kicks_you_out/

Adding this option would help make Brave even more minimalistic, which could attract more users from Chrome. Plus more option is alwasy better than it will suite more people. 

Thank you for reading, and please consider this request. Have a great day!

Brave Version: 1.73.89
Platform: Windows 11
Solution:  Option to Disable the Full-Screen 'X' Exit Button

**Comment by HornyPrivateGamer** [comment]
Any one please ?

**Comment by HornyPrivateGamer** [comment]
@brave-builds @brave-browser-releases

**Comment by jabarrera** [comment]
Yes, it also happening in GNU/Linux and this is potentially an issue with browser-based streaming gaming. I use Amazon Luna streaming gaming service, and playing with a mouse can be problematic because every time the cursor reaches the upper edge of the screen, the "X" appears, creating a risk of accidentally clicking it during the game.

Removing the annoying message was a great improvement, but addressing this issue is still necessary to fully resolve the problem.

Perhaps it would be better implemented as a site-specific option rather than a global setting. This way, you could configure the "disable exit fullscreen popup" option for specific sites only to avoid other problems.

![Image](https://github.com/user-attachments/assets/8b0e1c1e-7dd2-4313-b39c-015f5666d1a5)

Thanks!

**Comment by HornyPrivateGamer** [comment]
Halo Anyone ?

**Comment by Pbroc** [comment]
+1 

The current option in System settings for Dekstop on Windows 11 "Show full screen reminder to press Esc on exit" doesn't work when using F11.

So for example when I open a locally stored picture with Brave, I get that ugly pop up for 5 seconds~ over the subject of the picture.

Please fix thanks.
